src/app/communities/communities.service.spec.ts

```diff
@@ -234,4 +234,21 @@ describe('CommunitiesService', () => {
     req.flush(testCommunityResponse);
   }));
 
+  it('should make user leave a community', async(() => {
+
+    const communityId = 'd11235de-f13e-4fd6-b5d6-9c4c4e18aa4f';
+
+    service.leaveCommunity(communityId).subscribe((data: any) => {
+      expect(data).toEqual(null);
```

Comment:
The server will respond with a new DTO without a user in a member list. So I think it is better to expect here a community DTO and not `null` value.

Reply:
done